procedures/foreman/branch.md.erb

```diff
@@ -20,6 +20,11 @@
 * [ ] Add version <%= release %> to jobs in axes and/or combination filters, remove old ones (keep three)
 * [ ] Clone Debian nightly repos to <%= release %> using [copy/freight instructions](https://projects.theforeman.org/projects/foreman/wiki/Debian_Packaging#Project-sources)
 * [ ] Edit [foreman-packaging's PR template](https://github.com/theforeman/foreman-packaging/blob/master/.github/PULL_REQUEST_TEMPLATE.md) to add <%= release %> and remove the old release
+* [ ] Include Rails repository
+  * [ ] Add tfm-ror52 tag entries to Foreman tool_belt config to copy from main tfm-ror52 tag
```

Comment:
Should this be `Add foreman-rails-<%= release %>` tag entries? Or just explicitly `foreman-rails-<%= release %>-rhel7`